javascript/sdk/src/repl.ts

```diff
@@ -105,15 +105,17 @@ export class ReplExecResult {
     this.result = promise
     this.#resolve = resolve
     this.#reject = reject
+  }
 
-    this.output = {
+  get output(): { [Symbol.asyncIterator](): AsyncIterator<StandardOutput, void, unknown> } {
+    return {
       [Symbol.asyncIterator]: () => ({
         next: async () => {
           while (true) {
             const value = this.#buffer.shift()
             if (value) return { value, done: false }
 
-            if (this.#done) return { done: true }
+            if (this.#done) return { value: void 0, done: true }
```

Comment:
what does `void 0` do here? Is it different from `undefined`?

Reply:
oh weird, when i omitted the property i thought typescript was complaining that the type was `undefined` rather than `void`. but actually it's if i explicitly set it to `undefined`